config/routes.rb

```diff
@@ -2,6 +2,13 @@
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
 
   root to: 'home#index'
+
+  resources :admins, only: [] do
+    collection do
```

Comment:
Just curious here - why is this part in a collection?

Reply:
I could have just said namespace :admins and post :login, but I wanted a resource for later - so I can add index and show and all that stuff. I guess it might make more sense to put the `post :login` into a `member` block.

Reply:
I'll leave it for now, I don't have a better idea. Maybe you have a suggestion?

Reply:
I was just more curious if there was a significant reason for using a collection here vs simply:
```ruby
resources :admins, only: [] do
   post :login, controller: 'admin_sessions'
end
```